package.json

```diff
@@ -27,7 +27,7 @@
     "browserify": "browserify index.js --standalone faunadb -o dist/faunadb.js",
     "browserify-min": "browserify index.js --standalone faunadb | terser -c -m --keep-fnames --keep-classnames -o dist/faunadb-min.js",
     "prettify": "prettier --write \"{src,test}/**/*.{js,ts}\"",
-    "test": "jest --env=node --verbose=true --detectOpenHandles --forceExit ./test",
```

Comment:
What happens when we don't force the exit and detect open handles? Just curious why these went away, not a blocker.

Reply:
Nothing happens when we don't force exit. It's not necessary anymore. We fixed the cause of the hangup.
I think we removed the `--detectOpenHandles` because it was flaky, but @rts-rob would have to verify that claim.

Reply:
Correct - we added `--detectOpenHandles` as a part of #600. No harm in taking it out.

As Grey points out, the force exit was a patch over the actual fix, which this PR includes, and as such is no longer necessary.

Reply:
Sounds good. Go!